src/types/index.tsx

```diff
@@ -65,6 +62,14 @@ export type Class = {
   attendance: Attendance[];
 };
 
+export type Enrolment = {
+  id: number;
+  session: Session;
+  preferred_class: Class;
+  enrolled_class: Class;
```

Comment:
I don't believe the API response includes all the fields under the `Session` and `Class` types – we may want to make use of the `SessionListResponse` and `ClassListResponse` (defined in SessionAPI.ts) instead, which refer to the same serialization

Reply:
all these values are also [nullable](https://github.com/uwblueprint/project-read-backend/blob/9c8c7f146ba46865939afead7284e11002b15882/enrolments/models.py#L83-L99), so we should ensure that's captured in the type

Reply:
running into dependency cycles with using the ListRepsonse types btw 🌀 😞 
looking into it atm